src/components/SidePanes/CatalogInclusionPane.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import CatalogInclusionPane from './CatalogInclusionPane';
+
+describe('CatalogInclusionPane', () => {
+  const mockUuid = 'test-enterprise-id';
+  const mockInclusion = false;
+
+  it('correct toggle behavior', () => {
+    const wrapper = mount(<CatalogInclusionPane
+      courseUuid={mockUuid}
+      subInclusion={mockInclusion}
+    />);
+    wrapper.find(CatalogInclusionPane);
+    const title = wrapper.find('Enterprise Subscriptions');
+    expect(title);
+
+    const toggle = wrapper.find('.pgn__form-switch-input');
+    toggle.simulate('change', { target: { checked: false } });
+    expect(toggle.props().checked).toBe(false);
+  });
```

Comment:
This should have more test cases, such as draft behavior, error messages, toggle status, etc.